lib/composer.js

```diff
@@ -509,7 +509,7 @@ export default class Composer extends Disposable {
       builder.parseLogAndFdbFiles(jobState)
     }
 
-    const pattern = path.resolve(dir, jobState.getOutputDirectory(), `${jobState.getJobName() || name}*`)
+    const pattern = path.resolve(dir, jobState.getOutputDirectory(), `${jobState.getJobName() || name}.*`)
```

Comment:
There are a few automatically generated files that this pattern will miss. Specifically, BibTeX generates `{jobName}-blx.bib` and RevTeX generates `{jobName}Notes.bib`. Not sure if these are worth including or just relying on the existence of a fdb file instead.